### PR TITLE
Idiomatic Refactoring for GAX-PHP

### PR DESCRIPTION
**Pull request by bshaffer**
- Moves classes into separate files
- Fixes namespace for Jison classes
- Uses "Grpc" instead of "\Grpc"
- Removes globals
- Removes "get_object_vars"
- Removes "initClass"

### CODE REVIEW

File: src/Jison/Parser.php [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
- * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
- * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
- * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
- * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
- * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
- * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
- * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
- * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
- * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+/*
```
Comment (shinfan) [inline src/Jison/Parser.php]: Remove trailing space?

File: src/ApiCallable.php [hunk 2]
```diff
@@ -133,7 +133,7 @@ private static function setCustomHeader($callable, $headerDescriptor)
     }
 
     /**
-     * @param \Grpc\BaseStub $stub the gRPC stub to make calls through.
+     * @param Grpc\BaseStub $stub the gRPC stub to make calls through.
      * @param string $methodName the method name on the stub to call.
      * @param \Google\GAX\CallSettings $settings the call settings to use for this call.
```
Comment (shinfan) [inline src/ApiCallable.php]: Shall we remove "\" here as well?

Comment (bshaffer) [inline src/ApiCallable.php]: Yes! Good catch.

**Comment by michaelbausor** [comment]
LGTM
This is a huge improvement over what we had, thanks Brent!

**Pull request merged**